### PR TITLE
Add metadata fields to openapi-2-kong package.json

### PR DESCRIPTION
**Pull request by mikaello**
The metadata fields on https://www.npmjs.com/package/openapi-2-kong are a bit scarce, and it would be nice if these were improved:

image

### CODE REVIEW

File: packages/openapi-2-kong/package.json [hunk 1]
```diff
@@ -29,5 +29,8 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11"
   },
+  "repository": "https://github.com/Kong/insomnia/tree/develop/packages/openapi-2-kong",
+  "bugs": "https://github.com/Kong/insomnia/issues",
```
Comment (develohpanda) [inline packages/openapi-2-kong/package.json]: According to [the docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bugs), this needs to be the following:

```suggestion
  "bugs": {
    "url": "https://github.com/Kong/insomnia/issues"
  },
```

File: packages/openapi-2-kong/package.json [hunk 2]
```diff
@@ -29,5 +29,8 @@
...
+  "license": "MIT",
```
Comment (develohpanda) [inline packages/openapi-2-kong/package.json]: @nijikokun should the license be MIT or Apache2? Inso is set to Apache2.

**Review by develohpanda — COMMENTED** [review]
Thanks for doing this! One comment, and a question for @nijikokun

File: packages/openapi-2-kong/package.json (hunk 1 above)
Comment (mikaello) [reply packages/openapi-2-kong/package.json]: Thanks for looking up! I based my changes on an old convention, obviously outdated. Fixed now.

File: packages/openapi-2-kong/package.json (hunk 2 above)
Comment (mikaello) [reply packages/openapi-2-kong/package.json]: Just a comment to this question: if you set this to Apache2, I think you should change the licence that is distributed with openapi-2-kong, which is currently MIT: 

```shell
$ npm pack openapi-2-kong
$ tar xzf openapi-2-kong-2.2.24.tgz
$ cat package/LICENSE

MIT License

Copyright (c) Insomnia REST Client

Permission is hereby granted....
```

For reference why this came to my attention in the first place, see [discussion in Kong Nation](https://discuss.konghq.com/t/import-swagger-open-api-document-to-create-api-in-kong/510/18?u=michael_johansen).

**Review by develohpanda — APPROVED** [review]
🎉

File: packages/openapi-2-kong/package.json (hunk 2 above)
Comment (develohpanda) [reply packages/openapi-2-kong/package.json]: A [recent PR](https://github.com/Kong/insomnia/pull/3233) went in to set up license fields using feedback from the Legal team, so we can remove this field from this PR. I have created a separate internal ticket INS-658 in order to address your suggestion (with this package and all other packages).

```suggestion
```

File: packages/openapi-2-kong/package.json [hunk 3]
```diff
@@ -31,5 +31,14 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Kong/insomnia.git",
+    "directory": "packages/openapi-2-kong"
+  },
```
Comment (develohpanda) [inline packages/openapi-2-kong/package.json:38]: Nice improvement here! We should certainly update the other packages to follow this pattern, TIL.

**Review by reynolek — APPROVED** [review]
Approved from product perspective

**Pull request merged**